capstone/src/main/java/com/google/sps/servlets/student/StudentServlet.java

```diff
@@ -159,6 +171,31 @@ private static String getAnnouncementAsString(Entity announcement) {
             time);
     return fullAnnouncement;
   }
+
+  private Entity retrieveClub(String clubName, DatastoreService datastore) {
+    Query query =
+        new Query("Club")
+            .setFilter(
+                new FilterPredicate(Constants.PROPERTY_NAME, FilterOperator.EQUAL, clubName));
+    PreparedQuery results = datastore.prepare(query);
+    Entity club = ImmutableList.copyOf(results.asIterable()).get(0);
+    return club;
+  }
+
+  private void addItemToEntity(
+      Entity entity, DatastoreService datastore, String itemToAdd, String property) {
+    // Create emtpy ArrayList if property does not exist yet
+    ArrayList<String> generalList = new ArrayList<String>();
```

Comment:
nit: is it possible to just use normal List here?

Reply:
Done! Although, correct me if I'm wrong, I think the List still needs to be instantiated to add elements in the case that it's empty, which would be done with the ArrayList